pyproject.toml

```diff
@@ -28,7 +29,7 @@ consistent-sampler = {git = "https://github.com/votingworks/consistent_sampler.g
 scipy = "~1.6.0"
 Flask-SeaSurf = "^0.3.0"
 boto3 = "^1.20.8"
-r2b2 = {git = "https://github.com/gwexploratoryaudits/r2b2", rev = "128874b"}
```

Comment:
Pinning this commit broke for some reason, but there's only one commit after it on `main` in this repo, it isn't actively worked on, and it's only used for some experimental math that we don't use in real audits, so it seemed like a low risk workaround